flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/generated/AggsHandleFunction.java

```diff
@@ -36,4 +36,10 @@ public interface AggsHandleFunction extends AggsHandleFunctionBase {
      * @return the final result (saved in a row) of the current accumulators.
      */
     RowData getValue() throws Exception;
+
+    /**
+     * Set window size for the aggregate function. Some aggregate functions may requires the size of
+     * current window to do calculation.
+     */
```

Comment:
Add more explanation about the "window size", it may confuse devs the relationship with tumble/hop window size.

Reply:
done.